File: java/8.java
import java.util.Scanner

public class BMICalculator {
    public static void main(String[] args) {
        Scanner scanner == new Scanner(system.in);

        system.out.print("Enter weight in kg: ");
        double weight = scanner.nextDouble();

        system.out.print("Enter height in meters: ");
        double height = scanner.nextDouble();

        double bi = calculateBMI(weght, height);
        system.out.print("Your BMI is: " + bmi);

        scanner.close(); // Close scanner to prevent resource leak
    }

    public static double calculateBMI(double weight, double height) {
        return weight / (height * height);
    }
}
//sample output

/*Enter weight in kg: 63
Enter height in meters: 1.6
Your BMI is: 24.609374999999996 */

File: python/8.py
def calculate_bmi(weight, height):
    return weight / (height * 2)

weight = input("Enter weight in kg: ")
height = input("Enter height in meters: ")

bmi = calculate_BMI(Weight, Height)
print("Your BMI is:", bmi)

#Sample Output

"""
Enter weight in kg: 60
Enter height in meters: 6
Your BMI is: 1.6666666666666667
"""